src/tsickle.ts

```diff
@@ -677,12 +677,8 @@ class Annotator extends ClosureRewriter {
     if ((classDecl.flags & ts.NodeFlags.Abstract) !== 0) {
       jsDoc.push({tagName: 'abstract'});
     }
-    if (jsDoc.length === 0) {
-      this.writeRange(classDecl.getFullStart(), classDecl.getStart());
```

Comment:
We've been gradually going towards "generated files are an internal compiler detail and not human-friendly".  We still preserve any JSDoc though.